Presentation.Web/app/components/it-system/edit/tabs/it-system-usage-tab-interfaces.config.ts

```diff
@@ -0,0 +1,81 @@
+                interfaces: [
+                    "$http",
+                    $http => $http
+                        .get(
+                            "odata/LocalInterfaceTypes?$filter=IsLocallyAvailable eq true or IsObligatory&$orderby=Priority desc")
+                        .then(result => result.data.value)
+                ],
+                interfaceTypes: [
+                    "$http",
+                    $http => $http
+                        .get(
+                            "odata/LocalItInterfaceTypes?$filter=IsLocallyAvailable eq true or IsObligatory&$orderby=Priority desc")
+                        .then(result => result.data.value)
+                ],
+                methods: [
+                    "$http",
+                    $http => $http
+                        .get(
+                            "odata/LocalMethodTypes?$filter=IsLocallyAvailable eq true or IsObligatory&$orderby=Priority desc")
+                        .then(result => result.data.value)
+                ],
+                dataTypes: [
+                    "$http",
+                    $http => $http
+                        .get(
+                            "odata/LocalDataTypes?$filter=IsLocallyAvailable eq true or IsObligatory&$orderby=Priority desc")
+                        .then(result => result.data.value)
+                ],
+                frequencies: [
+                    "$http",
+                    $http => $http
+                        .get(
+                            "odata/LocalFrequencyTypes?$filter=IsLocallyAvailable eq true or IsObligatory&$orderby=Priority desc")
+                        .then(result => result.data.value)
+                ],
+                systemUsageId: [
+                    "$http", "itSystem", "user",
+                    ($http, itSystem, user) => $http
+                        .get(`api/itSystemUsage/?itSystemId=${itSystem.id}&organizationId=${user.currentOrganizationId}`)
+                        .then(systemUsageResult => systemUsageResult.data.response.id)
+                        .catch(response => {
+                            if (response.status === 404)
+                                return 0;
+                            throw "unexpected error when trying to get specific it system";
+                        })
+                ],
+                exhibits: [
+                    "$http", "itSystem", "user", "systemUsageId", ($http, itSystem, user, systemUsageId) =>
+                        $http.get(`api/exhibit/?interfaces=true&sysId=${itSystem.id}&orgId=${user.currentOrganizationId}`).then(result => {
+                            var interfaces = result.data.response;
+                            _.each(interfaces, (data: { exhibitedById; usage; }) => {
+                                if (systemUsageId !== 0) {
+                                    $http.get(`api/itInterfaceExhibitUsage/?usageId=${systemUsageId}&exhibitId=${data.exhibitedById}`).success(usageResult => {
```

Comment:
forkert filnavn

Reply:
Vi skal ikke have usage med her

Reply:
Usage skal ud af det her view. vi skal vise de udstillede IT-systemer på logisk neveau. Her får du kun den 1. usage og det giver ikke mening i det view der anvendes.